### DISCUSSION TITLE
Dataform Plugin Neovim

### DISCUSSION THREAD

**Discussion by magal1337** [dataform-co/dataform#1713] [Show and tell]
Hello guys!
For data engineers that like to use neovim, I created this dataform plugin with the following functionalities:
Compile dataform project when open .sqlx file first time within neovim session or when write a sqlx file
Compile dataform current model to sql script with bq cli validation (full load and incremental)
Go to reference sqlx file when line has the ${ref()} pattern
Run current dataform model (full or incremental)
Run current dataform model assertions
Run entire dataform project
Run dataform specific tag
Syntax highlighting for both sql and javascript blocks
If you liked the idea and want to contribute I’ll be more than happy to review your PRs :smiley:

https://github.com/magal1337/dataform.nvim

**Comment by LeonAndrade** [comment]
Nice!
